assets/css/theme-tugboat.css

```diff
@@ -0,0 +1,438 @@
+        --MENU-SECTION-ACTIVE-CATEGORY-color: #03384B; /* Color of active category text */
+        --MENU-SECTION-ACTIVE-CATEGORY-BG-color: #fff; /* Color of background for the active category (only) */
+        --MENU-SECTION-ACTIVE-CATEGORY-BORDER-color: #E8F0F5;
+
+        --MENU-VISITED-color: #33a1ff; /* Color of 'page visited' icons in menu */
+        --MENU-SECTION-HR-color: #20272b; /* Color of <hr> separator in menu */
+
+        /* Notice/Box colors for light mode */
+        --BOX-CAPTION-color: rgba(255, 255, 255, 1); /* text color of colored box titles */
+        --BOX-BG-color: rgba(255, 255, 255, 0.9); /* background color of colored boxes */
+        --BOX-TEXT-color: rgba(28, 66, 72, 1); /* text color of colored box content */
+        --BOX-GREY-color: rgba(232, 240, 245, 1); /* background color of grey boxes */
+
+        --CODE-theme: relearn-light; /* Light syntax highlighting */
+
+    }
+}
+
+/* Dark mode colors */
+@media (prefers-color-scheme: dark) {
+    :root {
+        --PRIMARY-color: #00B8E6;
+        --SECONDARY-color: #0098CB;
+        --ACCENT-color: #167ad0;
+
+        --MAIN-TEXT-color: #E8F0F5; /* Light text for dark background */
+        --MAIN-TITLES-TEXT-color: #F9FCFE; /* Lighter color for titles */
+        --MAIN-LINK-color: #00B8E6; /* Brighter blue for links on dark background */
+        --MAIN-LINK-HOVER-color: #33C4E8; /* Lighter hover color */
+        --MAIN-ANCHOR-color: #5BB4F3; /* Lighter anchor color */
+        --MAIN-BG-color: #1a1a1a; /* Dark content background */
+
+        --MENU-HEADER-BG-color: #1e1e1e; /* Darker header background */
+        --MENU-HEADER-BORDER-color: #2a3238; /* Border color */
+
+        --MENU-SEARCH-BG-color: #2a3238; /* Dark search field background */
+        --MENU-SEARCH-BOX-color: #2a3238; /* Dark search field border */
+        --MENU-SEARCH-BOX-ICONS-color: #E8F0F5; /* Light icons */
+
+        --MENU-SECTIONS-ACTIVE-BG-color: #111; /* Dark active section background */
+        --MENU-SECTIONS-BG-color: #1e1e1e; /* Dark menu background */
+        --MENU-SECTIONS-LINK-color: #B8CDD6; /* Light menu links */
+        --MENU-SECTIONS-LINK-HOVER-color: #00B8E6; /* Bright hover color */
+        --MENU-SECTION-ACTIVE-CATEGORY-color: #fff; /* Active category color */
+        --MENU-SECTION-ACTIVE-CATEGORY-BG-color: #111; /* Active category background */
+        --MENU-SECTION-ACTIVE-CATEGORY-BORDER-color: #1e1e1e;
+
+        --MENU-VISITED-color: #5BB4F3; /* Visited page icons */
+        --MENU-SECTION-HR-color: #4a5258; /* HR separator color */
+
+        /* Notice/Box colors for dark mode */
+        --BOX-CAPTION-color: rgba(232, 240, 245, 1); /* text color of colored box titles */
+        --BOX-BG-color: rgba(42, 50, 56, 0.9); /* background color of colored boxes */
+        --BOX-TEXT-color: rgba(232, 240, 245, 1); /* text color of colored box content */
+        --BOX-GREY-color: rgba(74, 82, 88, 1); /* background color of grey boxes */
+
```

Comment:
```suggestion
-- VARIABLE-B0X-color: #0098cb
```
![Screenshot 2025-06-13 at 8 02 32 AM](https://github.com/user-attachments/assets/e648a4b9-fb0d-4670-986d-07008f00ceac)